.vscodeignore

```diff
@@ -4,7 +4,8 @@
 
 # Include the following:
 !images/
-!dist/
+!dist/**/*.js
+!dist/**/*.css
```

Comment:
This seems odd. As part of the plugin packaging, this includes both the Common JS entry point and an ESM export, but not the `.d.ts` files that would help users consume those. If the intention is to include in the plugin package only the relevant JS files, then I believe that only `.cjs.js` should be included; if the goal is to include all the files necessary to consume the plugin as an NPM-like dependency, then `.d.ts` files (and maybe the .esm.js file?) should be included, as well.

Reply:
Hi @colin-grant-work, I included `d.ts` files to npm package and consider VSIX package as a runtime environment and exclude the `d.ts` files as well as `js.map` files from the VSIX package. Does this sounds logical to you?